app/models/kudo.rb

```diff
@@ -3,4 +3,6 @@ class Kudo < ApplicationRecord
   validates :content, presence: true
   belongs_to :giver, class_name: 'Employee' # , dependent: :kudo
   belongs_to :receiver, class_name: 'Employee'
+  belongs_to :giver, class_name: 'Admin' # , dependent: :kudo
+  belongs_to :receiver, class_name: 'Admin'
```

Comment:
this is a mistake to have multiple associations named the same way
also again, why do you create an association between kudo and admin?